server/bin/pbench-audit-server.sh

```diff
@@ -772,13 +772,17 @@ fi
 
 log_finish
 
-# send it
-subj="$PROG.$TS($PBENCH_ENV)"
-cat << EOF > ${index_content}
+subj="$PROG.$TS ($PBENCH_ENV)"
+if [[ -s ${report} ]]; then
+    cat << EOF > ${index_content}
```

Comment:
Consider using `<<-` where you have `<<` -- this allows you to indent lines 778-780, which is nicer.

Reply:
Alternatively, you could use `<<<`, which would be much nicer, still.  (You might have to use a second command to insert the blank line, but that would still be better than the "here-document".)

Reply:
Um, wait...you're trying to put the contents of a string variable plus a couple of carriage returns into a file?  That sounds like a job for `echo` (or your favored `printf`)!
```
if [[ -s ${report} ]]; then
    echo -e "$subj\n" > ${index_content}
    cat ${report} >> ${index_content}
```

Reply:
But this script is going to be obsolete on the `main` branch soon, and we don't need to do this level of improvement on the legacy branch, `b0.69`.